plugins/woocommerce-blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/frontend.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * External dependencies
+ */
+import { store } from '@woocommerce/interactivity';
```

Comment:
And this one too: :slightly_smiling_face: 
```suggestion
import { store } from '@wordpress/interactivity';
```